src/main/java/quickndirty/minisurveymonkey/RangeQuestion.java

```diff
@@ -0,0 +1,48 @@
+package quickndirty.minisurveymonkey;
+
+import javax.persistence.Entity;
+
+@Entity
+public class RangeQuestion extends Question{
+	private int min;
+	private int max;
+	
+	public RangeQuestion(){
+		type = QuestionType.NUMBER;
+	}
+
+	public RangeQuestion(String Prompt, int min, int max) {
+		type = QuestionType.NUMBER;
+		this.prompt = Prompt;
+		this.min = min;
+		this.max = max;
+		
+		
+	}
+	
+	public boolean addResponse(RangeResponse r) {
+		if(r.getAnswer() < min || r.getAnswer() > max) {
+			return false;
+		}else {
+			Response q = (Response)r;
```

Comment:
Is this line needed?

Reply:
That was actually a typo, I'll fix it, thanks.